node-tests/acceptance/end-to-end-test.js

```diff
@@ -75,7 +75,7 @@ describe('end-to-end', function() {
       let { name: tmpDir } = tmp.dirSync();
       process.chdir(tmpDir);
 
-      return ember('new', 'ee-test-app', '--yarn').then(() => {
+      return ember('new', 'ee-test-app', '--yarn', '--skip-git', '--no-welcome').then(() => {
```

Comment:
I forgot to mention this in the commit message, but I took the liberty of adding these parameters here since they shouldn't hurt anything (we're not using the git repository created for the new app nor the welcome page), and skipping these should improve test performance a little bit (less file I/O by skipping repo and fewer messages cluttering the log by not having the welcome page package installed).